tools/tpm/policy_sign.c

```diff
@@ -29,6 +29,7 @@
 #include <wolfssl/wolfcrypt/ecc.h>
 #include <wolfssl/wolfcrypt/hash.h>
 #include <wolftpm/tpm2_wrap.h>
+#include <wolfssl/error-ssl.h>
```

Comment:
We don't want to include any TLS/SSL headers.
Instead of adding this header please replace `INPUT_SIZE_E` with `BUFFER_E`. 
See http://github.com/wolfSSL/wolftpm/pull/308